src/js/components/List/doc.js

```diff
@@ -47,6 +47,10 @@ export const doc = List => {
 
   DocumentedList.propTypes = {
     ...genericProps,
+    action: PropTypes.func.description(
+      `Action will be something for the user to click on and interact
+       with within the list component`,
```

Comment:
Needs to be a little more clear as @IanKBovard suggested.
what about a different opening, "The interactive element property for the user to..."? thoughts?

Reply:
I wonder if we should change the Action prop name like suggested above then..

Even from looking at the code I was not clear on the `action` prop